tests/deps.ts

```diff
@@ -9,4 +9,4 @@ export {
   Context,
   session,
   type SessionFlavor,
-} from "https://lib.deno.dev/x/grammy@1.x/mod.ts";
+} from "https://lib.deno.dev/x/grammy@>=1.11/mod.ts";
```

Comment:
Can I revert this 😄? It'll break tests dependency transforming. Thats why CI failed. lib.deno.dev/x/grammy@1.x will redirect to >=1.11 anyways. So...

Reply:
hmm why?

Reply:
Tests currently uses a [test lib](https://github.com/dcdunkan/tests) that I wrote. It has no node package, so dnt will transform it to node as well. So, while transforming, imports needs mapping to node packages. So, currently it only has mappings for 1.x url imports (see scripts/dnt.ts).

It can be fixed by just adding additional (basically, duplicate) mapping for >=1.11. But its better to use 1.x. Because as of now, 1.x import is not going to resolve to <1.x. So, basically 1.x = >=1.11.

Reply:
Okay 👌

Reply:
Thanks!

Reply:
CI failed again :(